src/monitor-plan-workspace/monitor-plan.controller.ts

```diff
@@ -89,7 +89,7 @@ export class MonitorPlanWorkspaceController {
   ): Promise<any> {
     await this.mpChecksService.runChecks(plan);
     await this.importChecksService.runImportChecks(plan);
-    return await this.service.importMpPlan(plan, user.userId, draft);
+    return await this.service.importMpPlan(plan, user.userId, true); // XXX: Use draft parameter when done testing
```

Comment:
Did you meant to leave this in?

Reply:
I did not mean to leave that in, but I did remove it in [7469d14](https://github.com/US-EPA-CAMD/easey-monitor-plan-api/pull/565/commits/7469d14e301f5da4a5bbd4670ae92cf36b6ee056).